### ISSUE TITLE
synergia build fails: Unknown configure option "fftw3/make_custom_parallel".

### ISSUE THREAD

**Issue by robnagler** [radiasoft/download#49]

**Comment by robnagler** [comment]
Numerous other errors. It seems chef-libs doesn't build in parallel, getting random errors like:

```txt
[  1%] Built target create_includedir
/home/vagrant/src/fnalacceleratormodeling/contract-synergia2/build/chef-libs/basic_toolkit/src/erf.cc:39:10: fatal error: basic_toolkit/iosetup.h: No such file or directory
 #include <basic_toolkit/iosetup.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

and

```txt
[  1%] Built target create_includedir
/home/vagrant/src/fnalacceleratormodeling/contract-synergia2/build/chef-libs/basic_toolkit/src/Barnacle.cc:55:10: fatal error: basic_toolkit/Barnacle.h: No such file or directory
 #include <basic_toolkit/Barnacle.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Another error is that boost_python27 is being required, but it isn't available until boost 1.67. Fedora 27 is boost 1.64 and the internal boost is 1.61.

**Comment by robnagler** [comment]
FTR: 

* boost_python https://bitbucket.org/fnalacceleratormodeling/synergia2/pull-requests/1
* hdf5 https://bitbucket.org/fnalacceleratormodeling/contract-synergia2/pull-requests/1
